src/write_redis.c

```diff
@@ -125,6 +126,16 @@ static int wr_write(const data_set_t *ds, /* {{{ */
       freeReplyObject(rr);
   }
 
+  if (node->max_set_duration > 0) {
+    rr = redisCommand(node->conn, "ZREMRANGEBYSCORE %s -1 (%d", key, 
```

Comment:
The not-matched open parenthesis in `(%d` looks suspicious. If this in on purpose, please add a comment explaining it.

Reply:
@octo, Thank you for your comment! '(%d' means 'less than %d' of redis score. I will update it soon.

Reply:
@octo I've added comments in latest diff. Thanks again!